styles.js

```diff
@@ -71,6 +78,6 @@ export default createGlobalStyle`
     width: 200px;
     margin: auto;
   }
-`;
 
-// #00a0a0 darker petrol
+
+`;
```

Comment:
We agreed on not using classes to styled the apps. Its a requirement to use styled components. Please transform these element, class and id selectors into styled components. The globalstyles are only meant for the universal selector `*{...}`, the @fontface and maybe themes for the dark and light modes. everything else needs to be a styled component :)